Bookish.Web/Startup.cs

```diff
@@ -65,16 +65,19 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
                 app.UseDatabaseErrorPage();
+                app.UseExceptionHandler("/StatusCode/500");
             }
             else
             {
-                app.UseExceptionHandler("/Home/Error");
+                app.UseExceptionHandler("/StatusCode/{0}");
```

Comment:
I'd just do this outside the if as you want the same behaviour with and without dev mode 🙂